src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -0,0 +1,527 @@
+                  onChange={this.onInputChange}
+                />
+              </FormGroup>
+
+              <FormGroup legendText="Image Resource (optional)">
+                <TextInput
+                  id="image-name-text-input"
+                  labelText="Name"
+                  placeholder="docker-image"
+                  name="imageName"
+                  invalidText={resourceNameInvalidText}
+                  invalid={this.state.imageName.invalid}
+                  value={this.state.imageName.value}
+                  onChange={this.onInputChange}
+                />
+                <TextInput
+                  id="image-registry-text-input"
+                  labelText="Registry"
+                  placeholder="dockerhubusername"
+                  name="imageRegistryName"
+                  invalidText={noSpacesInvalidText}
+                  invalid={this.state.imageRegistryName.invalid}
+                  value={this.state.imageRegistryName.value}
+                  onChange={this.onInputChange}
+                />
+                <TextInput
+                  id="image-repo-text-input"
+                  labelText="Repository"
+                  placeholder="reponame"
+                  name="imageRepoName"
+                  invalidText={noSpacesInvalidText}
+                  invalid={this.state.imageRepoName.invalid}
+                  value={this.state.imageRepoName.value}
+                  onChange={this.onInputChange}
+                />
+              </FormGroup>
+
+              <FormGroup legendText="Helm (optional)">
+                <Toggle
+                  id="helm-pipeline-toggle"
+                  data-testid="helm-pipeline-toggle"
+                  labelText="Toggle On when using a Helm pipeline"
+                  name="helmPipeline"
+                  toggled={this.state.helmPipeline.value}
+                  onToggle={this.onHelmToggleChange}
+                />
+                <TextInput
+                  id="helm-secret-text-input"
+                  labelText="Secret"
+                  placeholder="helm-secret"
+                  name="helmSecret"
+                  invalidText={resourceNameInvalidText}
+                  invalid={this.state.helmSecret.invalid}
+                  value={this.state.helmSecret.value}
+                  onChange={this.onInputChange}
+                />
+              </FormGroup>
+            </ModalBody>
+            <ModalFooter>
+              <Button kind="secondary" onClick={this.onClose}>
+                Cancel
+              </Button>
+              <Button kind="primary" type="submit">
+                Submit
+              </Button>
+            </ModalFooter>
+          </ComposedModal>
+        </Form>
+      </>
+    );
+  }
+}
+
+CreatePipelineRun.defaultProps = {
+  open: false,
+  onClose: () => {},
+  onSuccess: () => {}
+};
+
+const mapStateToProps = state => {
+  return {
+    errorMessage: getPipelineRunsErrorMessage(state),
```

Comment:
These aren't used. Id say make call directly from the component and handle the error here